src/hotspot/os/aix/os_aix.inline.hpp

```diff
@@ -31,6 +31,11 @@
 #include "runtime/os.hpp"
 #include "os_posix.inline.hpp"
 
+// Information about the protection of the page at address '0' on this os.
+inline bool os::zero_page_read_protected() {
+  return false;
+}
```

Comment:
I have the feeling many of these functions are really just global constants and that the coding could be simplified further in followup RFEs.

Reply:
Yeah lets do more clean up in follow-up RFEs.